JS/Lesson24/tugba/filter.js

```diff
@@ -260,11 +263,16 @@ const productsExercise6 = [
 		quantity: 14,
 	},
 ];
+const inStockProducts = productsExercise6.filter(product => product.quantity > 0);
+
+console.log(inStockProducts);
 
 // Exercise 7:
 // Given an array of numbers, filter out the numbers that are divisible by 3.
 const numbersExercise7 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+const numbersDivisibleBy3 = numbersExercise7.filter(number => number % 3 === 0);
```

Comment:
```suggestion
const numbersDivisibleBy3 = numbersExercise7.filter(number => number % 3 !== 0);
``` 
since the exercise says "filter out the numbers that are divisible by 3."